dist/index.d.ts

```diff
@@ -0,0 +1,4 @@
+import { StoreApi } from 'zustand';

```

Comment:
The dist folder should be excluded as it's the built code (probably it's missing in .gitignore)

Reply:
Oh yeah, sorry, forgot to revert that; weirdly, to use the package from a forked repo the dist needs to be pushed to that forked repo - otherwise it doesn't work. This is not the case when using it from your original repo - didn't find out why that is. Thx for fixing it 🙏